index.html

```diff
@@ -45,10 +45,246 @@ <h1 class="hero__heading poppins">
             </ul>
         </div>
 
+
+    </section>
+
+    <!-- card works -->
+    <section id="project">
+
+        <div class="project">
+
+            <img src="images/Snapshoot_Portfolio.svg" alt="">
+
+
+            <h1 class="project__heading poppins">
+                Tonic
+            </h1>
+
+            <ul class="project__path poppins">
+                <li class="project__path__name poppins">CANOPY</li>
+                <li class="project__path__type poppins">BACK END DEV</li>
+                <li class="project__path__type poppins">2015</li>
+            </ul>
+
+            <p class="project__desc poppins">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="project__language">
+                <li class="poppins">html</li>
+                <li class="poppins">css</li>
+                <li class="poppins">javaScript</li>
+            </ul>
+
+            <div>
+                <button type='button' 
+                class="project__button poppins">
+                    see Project
+                </button>
+            </div>
+
+
+
+
+        </div>
+
+        <div class="project">
+
+            <img src="images/multipost_stories.svg" alt="">
+
```

Comment:
Kindly specify a value for the `alt` attribute. It's necessary because that is what the user will see should the image not load properly. Also, screenreaders and other machines will thank you for that.